server/services/v1/query_runner.go

```diff
@@ -935,6 +937,11 @@ func (runner *CollectionQueryRunner) Run(ctx context.Context, tx transaction.Tx,
 			return nil, ctx, err
 		}
 
+		namespace, err := request.GetNamespace(ctx)
```

Comment:
Can we have a wrapper for it?
so we don't need to error check every time.

Reply:
Makes sense, moved it with other namespace related tagging helper.